editor/src/clj/editor/library.clj

```diff
@@ -108,13 +108,21 @@
 
 (defn- make-basic-auth-headers
   [^String user-info]
-  {"Authorization" (format "Basic %s" (str->b64 user-info))})
+  (let [user-info-parts (str/split user-info #":")
+        username (get user-info-parts 0)
+        password (get user-info-parts 1)]
+          (if (and (str/starts-with? password "__") (str/ends-with? password "__"))
+            (let [password-env-key (subs password 2 (- (count password) 2))
+                  password-env-value (or (System/getenv password-env-key) password)]
+                     {"Authorization" (format "Basic %s" (str->b64 (str username ":" password-env-value)))})
```

Comment:
The indentation in the `let` here and on line 114 is too big, can you please reformat these lines?